posts/2021/09/re-engineering-cicd-pipelines-for-scipy.md

```diff
@@ -0,0 +1,211 @@
+<!--
+.. title: Re-Engineering CI/CD pipelines for SciPy
+.. slug: re-engineering-cicd-pipelines-for-scipy
+.. date: 2021-09-29 05:51:58 UTC-05:00
+.. author: Harsh Mishra
+.. tags: SciPy, cicd, github-actions, internship-2021
+.. category: 
+.. link: 
+.. description: 
+.. type: text
+.. previewimage: /images/2021/10/re-engineering-cicd-pipelines-for-scipyre-engineering-ci-cd-scipy.png
+-->
+
+In this blog post I talk about the projects and my work during my internship at Quansight Labs. My efforts were geared towards re-engineering CI/CD pipelines for SciPy to make them more efficient to use with GitHub Actions. I also talk about the milestones that I achieved, along with the associated learnings and improvements that I made.
+
+This blog post would assume a basic understanding of CI/CD and GitHub Actions. I will also assume a basic understanding of Python and the SciPy ecosystem.
+
+<p align="center">
+    <img
+     alt="The project will enable the developers to test out the all-new Meson build system on the GitHub Actions CI and lays down the further scope for developing an entire CI matrix for build, test and release for SciPy."
```

Comment:
The alt-text should include a description of the image. In this case, it could be something like "The Quansight Labs and SciPy logos side by side." There is probably already a description of those logos somewhere, we could even link to that I guess.